docs/docs/adrs/adr-013-equivocation-slashing.md

```diff
@@ -0,0 +1,169 @@
+
+
+
+#### Clock drift between the provider and the consumer chain
+(Still thinking on this but adding here as a first pointer to some ideas)
+
+Conceptually, we have 2 different chains and there’s no guarantee that the clock drift between them is bounded.
+One could have BFT time of 2023 on one chain and on the other chain a BFT time of 2013. 
+Nevertheless, the chains communicate over IBC and hence some timing constraints are imposed by IBC otherwise the chains could not communicate (e.g., light clients could expire).
+IBC clients contain a `maxClockDrift` parameter but this is [only used](https://github.com/cometbft/cometbft/blob/v0.37.0/light/verifier.go#L176)
+to reject headers that come slightly from the future. For example, assume a chain `A` that has a light client `lc` that tracks chain `B` and that has `maxClockDrift` of 10 seconds. If chain `A` has time `12:58:31` and then `lc` receives a header with time `12:58:42` (that is 11 seconds from the future), then `lc` would reject his client update.
+However, even with this `maxClockDrift` constraint we still do not know the time at the chain `B`, e.g., `B`'s BFT time could be 5 years ahead compared to chain's `A` BFT time.
+
+(TODO: add more on the `trustingPeriod`) only tells us from the provider side that we have received a message from the other chain in the last `trustingPeriod` (2 weeks in Cosmos Hub) before ... so doesn't add much either ...
+
+We currently investigate on whether we can bound the clock drift between the provider and the consumer chain.
+
+## Proposed solution
+As a first iteration, in order to implement the slashing functionality as soon as possible, we propose an aggressive (i.e., would slash more delegators than in the single-chain case) slashing  approach because this allows for a straight-forward implementation.
+In this approach, at the moment we receive evidence, at _evidence height_, we:
+1. slash all the undelegations and redelegations using `slashFactor`;
+2. slash all delegations using as voting `power` the sum of the voting power of the misbehaving validator and the power of all the ongoing undelegations and redelegations.
+
+**Evidence expiration:** Additionally, in this approach and because we cannot infer the actual time of the evidence (i.e., timestamp in evidence could contain anything), we do not consider _evidence expiration_ and hence old evidence is never ignored (e.g., we would act on 3 year-old evidence).
+
+### Implementation
+The aggressive approach allows for multiple simplifications. Specifically, we intend to introduce the following logic in the [HandleConsumerDoubleVoting](https://github.com/cosmos/interchain-security/pull/1232) method:
```

Comment:
```suggestion
The following logic needs to be added to the [HandleConsumerDoubleVoting](https://github.com/cosmos/interchain-security/pull/1232) method:
```

Reply:
What about light client attacks? Do we need to differentiate?

Reply:
The described approach (i.e,. adding code `HandleConsumerDoubleVoting`) won't be useful for light client attacks as is. To slash for light client attacks we would need to do the same in  `HandleConsumerMisbehaviour` in response to `MsgSubmitConsumerMisbehaviour` messages.

However, I was thinking of this ADR specifically for double voting misbehaviour and how to slash in those cases.
It's not clear to me exactly how we verify (and punish for) light client attacks today since the staking module [Slash](https://github.com/cosmos/cosmos-sdk/blob/main/x/staking/keeper/slash.go#L193) is for equivocations. I would need to investigate this more in order to answer your questions.